Assets/Scripts/Buddy/BuddyStats.cs

```diff
@@ -212,6 +212,8 @@ public void AdjustHappiness( float deltaHappiness )
 			_happiness = 0;
 		}
 
+		actor.animator.SetFloat( "happiness", _happiness );
```

Comment:
Yeah it does assume normalized. If it goes over one it won't cause errors, but the buddies will seem extra happy.